### PR TITLE
docs enhancements

### PR DESCRIPTION
**Pull request by aboedo**
Fixed a few more symbol links, and cleaned up docs in general.

### CODE REVIEW

File: Purchases/Purchasing/Purchases.swift [hunk 1]
```diff
@@ -463,114 +462,127 @@ extension Purchases {
...
     @objc public func setDisplayName(_ displayName: String?) {
         subscriberAttributesManager.setDisplayName(displayName, appUserID: appUserID)
     }
 
     /**
-     * Subscriber attribute associated with the push token for the user
+     * Subscriber attribute associated with the push token for the user.
      *
-     * - Parameter pushToken: nil will delete the subscriber attribute.
+     * - SeeAlso: [Subscriber attributes](https://docs.revenuecat.com/docs/subscriber-attributes)
```
Comment (aboedo) [inline Purchases/Purchasing/Purchases.swift:497]: These `SeeAlso` don't seem to be working correctly for now. It looks like maybe Xcode doesn't support them anymore? 
I'm going to take a stab at fixing these in a separate PR, but for now, we're just following the standard notation

File: Purchases/Purchasing/Purchases.swift [hunk 2]
```diff
@@ -45,11 +45,10 @@ public typealias DeferredPromotionalPurchaseBlock = (@escaping PurchaseCompleted
     )
 #endif
 
-    /// Returns the already configured instance of `Purchases`.
-    /// - Warning: this method will crash with `fatalError` if `Purchases` has not been
-    ///            initialized through `configure()`.
-    ///            If there's a chance that may have not happened yet, you can use ``isConfigured``
-    ///            to check if it's safe to call.
```
Comment (aboedo) [inline Purchases/Purchasing/Purchases.swift:52]: indentation was unnecessary here

File: Purchases/Purchasing/Purchases.swift [hunk 3]
```diff
@@ -1113,18 +1135,18 @@ public extension Purchases {
      * the introductory price. This also applies to trials (trials are considered a type of introductory pricing).
      * [iOS Introductory  Offers](https://docs.revenuecat.com/docs/ios-subscription-offers).
      *
-     * - Note: If you're looking to use Promotional Offers use instead,
+     * - Note: If you're looking to use Promotional Offers instead,
      * use ``Purchases/checkPromotionalDiscountEligibility(forProductDiscount:product:completion:)``.
      *
      * - Note: Subscription groups are automatically collected for determining eligibility. If RevenueCat can't
-     * definitively compute the eligibilty, most likely because of missing group information, it will return
+     * definitively compute the eligibility, most likely because of missing group information, it will return
```
Comment (aboedo) [inline Purchases/Purchasing/Purchases.swift:1142]: Xcode spell check for the win

File: Purchases/Purchasing/Purchases.swift [hunk 4]
```diff
@@ -463,114 +463,127 @@ extension Purchases {
...
     /**
-     * Subscriber attribute associated with the OneSignal Player ID for the user
-     * Required for the RevenueCat OneSignal integration
+     * Subscriber attribute associated with the OneSignal Player ID for the user.
+     * Required for the RevenueCat OneSignal integration.
      *
-     * - Parameter onesignalID: nil will delete the subscriber attribute
+     * - SeeAlso:
+     * [mParticle RevenueCat Integration](https://docs.revenuecat.com/docs/onesignal)
+     * - Parameter onesignalID: Empty String or `nil` will delete the subscriber attribute..
```
Comment (aboedo) [inline Purchases/Purchasing/Purchases.swift:555]: oops, copy/pasta issue here, fixed in #1253

**Review by NachoSoto — APPROVED** [review]
So many nice improvements 🎉

**Pull request merged**